### PR TITLE
[MRESOLVER-367] Update Redisson and related documentation

### PR DESCRIPTION
**Pull request by cstamas**
This PR updates only redisson module: redisson dependency, simplifies installation by providing a "bundle" ZIP and updates site page.

---

https://issues.apache.org/jira/browse/MRESOLVER-367

### CODE REVIEW

**Review by gnodet — APPROVED** [review]
It could be worth creating a separate JIRA for this one as the new installation steps for Redisson is a nice new feature !

File: maven-resolver-named-locks-redisson/pom.xml [hunk 1]
```diff
@@ -85,7 +89,7 @@
     <dependency>
       <groupId>javax.inject</groupId>
       <artifactId>javax.inject</artifactId>
-      <optional>true</optional>
+      <scope>provided</scope>
     </dependency>
```
Comment (michael-o) [inline maven-resolver-named-locks-redisson/pom.xml:93]: Same question with scopes here as well.

**Comment by cstamas** [comment]
Created MRESOLVER-367 for this.

**Comment by michael-o** [comment]
WTF? You didn't even give me the chance to verify this.

**Comment by cstamas** [comment]
I did ping you in slack and notified you this morning about my intent. 

IMHO, this change can be tested just fine while merged as well, worst outcome is you find anything to fix, and create PR to fix it. Or potentially extra commit in history is what would be the problem?

**Comment by michael-o** [comment]
> I did ping you in slack and notified you this morning about my intent.
> 
> IMHO, this change can be tested just fine while merged as well, worst outcome is you find anything to fix, and create PR to fix it. Or potentially extra commit in history is what would be the problem?

..and you expect me to react on the ring immediate like a dog? C'mon!

**Comment by cstamas** [comment]
No I do not.

But similarly, you should not expect me to hang on a wall in waiting for "will do this then and then" to happen. IMHO, nobody should "reserve" or appropriate the privilege of PR for themselves, it's free for all. We all have time constraints, and the PR was approved already.

**Comment by olamy** [comment]
maybe as a matter of courtesy or respect to someone from the community who take the time to look at the PR and said `This I will test this week.` you could wait 2/3 days. 
There is no such rush to get this merged ASAP.

**Comment by cstamas** [comment]
Yes, I agree here as well: this move was maybe rude, or at least "not nice" toward Michael. I apologize for that.

But in short, if you have time, do what you have to do, otherwise step aside, and let others do it, like it happened in this case. No rush expected from anyone.

**Comment by cstamas** [comment]
@michael-o Did testing of this reveal any problem?

**Comment by michael-o** [comment]
> @michael-o Did testing of this reveal any problem?

No, not yet. The window I had was gone this week. Maybe I will hve one next week.

**Comment by jira-importer** [comment]
Resolve #1039

**Comment by jira-importer** [comment]
Resolve #1040

**Comment by jira-importer** [comment]
Resolve #1040

**Pull request merged**